modules/apps/object/object-service/src/main/java/com/liferay/object/internal/scope/SiteObjectScopeProviderImpl.java

```diff
@@ -79,7 +79,7 @@ public boolean isGroupAware() {
 	public boolean isValidGroupId(long groupId) {
 		Group group = _groupLocalService.fetchGroup(groupId);
 
-		if ((group != null) && group.isSite()) {
+		if ((group != null) && (group.isSite() || group.isUserGroup())) {
```

Comment:
@devpedrotavares did you check with the product team if we can support this new type of site?

Reply:
Yes, they verified the bug